gcd/src/main/java/org/spine3/sample/gae/Application.java

```diff
@@ -48,7 +53,10 @@ public Application(DatastoreStorageFactory storageFactory) {
      */
     public static void main(String[] args) {
         final LocalDatastoreStorageFactory storageFactory = LocalDatastoreStorageFactory.getDefaultInstance();
+        storageFactory.getOptions().setEventIteratorPageSize(ITERATOR_PAGE_SIZE);
+        storageFactory.setUp();
```

Comment:
@alexander-yevsyukov, I remember we had `init` method in the `StorageFactory`, but it was removed. Maybe, we should add it if it is necessary here?